spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -716,19 +604,22 @@ function refundTokens(packet: Packet) {
 // If an error occurs further down the line, the state changes
 // on this chain must be reverted before sending back the error acknowledgement
 // to ensure atomic packet forwarding
-function revertInFlightChanges(sentPacket: Packet, receivedPacket: Packet) {
-  forwardingAddress = channelForwardingAddress[receivedPacket.destChannel]
-  reverseEscrow = channelEscrowAddresses[receivedPacket.destChannel]
+function revertInFlightChanges(
```

Comment:
This function was incorrectly converted. Note that it originally took in **two** different packets and now you only have a single destChannelId and payload. You need to still take in the forwarded packet info and the original received packet info